app/models/user.rb

```diff
@@ -170,7 +170,7 @@ def sheerid_supported?
   def is_test?
     !!is_test
   end
-  
+
   def is_anonymous?
     false
```

Comment:
this seems odd to just return false. is it a default? or a placeholder for something in the future?  if it's just false, when why a separate method?

Reply:
It's actually very important that this method is defined and that it returns false. We use it to determine if a user is logged in or not. A logged in user returns `true` for `is_anonymous?`.